package.json

```diff
@@ -8,8 +8,9 @@
     "compress": "node tasks/compress.js",
     "pack": "node tasks/pack.js",
     "build": "run-s clean compress pack",
+    "preversion": "git pull --tags",
```

Comment:
Consider using `git fetch --tags` which should be equivalent, but maybe less confusing. I think of pull as a fetch and merge, even though it doesn't do that with the `--tags` option, a fetch better describes its behaviour.

Reply:
Certainly I feel good with `git fetch --tags`. It is doubtful whether can list the tags of the tracking branch with `git tag` ... I will check once.